addons/account_payment/models/account_payment.py

```diff
@@ -132,7 +132,10 @@ def action_post(self):
         )
         # creating the transaction require to access data on payment providers, not always accessible to users
         # able to create payments
-        transactions = payments_need_tx.sudo()._create_payment_transaction()
+        extra_create_values = {}
+        if self._context.get('active_model') == 'account.move':
+            extra_create_values['invoice_ids'] = self._context.get('active_ids')
```

Comment:
`invoice_ids` is a X2M -> `[Command.set(self._context.get('active_ids)]`

Reply:
I'm not sure if the [] default is really necessary here.

Reply:
Done

> I'm not sure if the [] default is really necessary here.

It is required to prevent a traceback with None value.

Reply:
Yeah, that's why I put it there first, but now I wonder if the ORM doesn't set `[]` as default for the `active_ids` key... Anyway, it's clearer like that and not worth spending time on it :)